config/locales/en.yml

```diff
@@ -50,6 +50,7 @@ en:
   withdrawn_participant: Cannot perform actions on a withdrawn participant
   declaration_on_incorrect_state: "The declaration on withdrawn or deferred participant can not be accepted"
   declaration_already_voided: "Declaration is already voided"
+  declaration_not_voidable: "Declaration not voidable"
```

Comment:
not entirely sure but I think the format of the error message should be something the follows that convention:

`The property '#/state' is not voidable`

?

Reply:
where is that convention found? I'm only keeping parity with what's there now. I feel we'd want to change all those in one go rather than have one snowflaked error message

Reply:
I was under the impression this was the format used for properties, there are plenty of examples throughout the yaml file, i.e line 56, 57. 
I'll add error message format consistency to our spreadsheet of API improvements.